src/parsing/lpParser.mly

```diff
@@ -325,13 +306,11 @@ aterm:
   | L_CU_BRACKET t=term R_CU_BRACKET { make_pos $sloc (P_Expl(t)) }
   | n=INT { make_pos $sloc (P_NLit(n)) }
 
-sterm:
-  | t=sterm u=aterm { make_pos $sloc (P_Appl(t,u)) }
+saterm:
```

Comment:
Why `saterm`? I mean `sterm` and `aterm` refer to "atomic" terms and "symbolic" terms, is there a meaning being "saterm"?

Reply:
I thought sterm = sequence of term's. So saterm = sequence of aterm's.